Dockerfile

```diff
@@ -1,196 +1,249 @@
-    md5sum /config/config.yaml > /config/build_installed && \
-    chmod +x /docker-entrypoint.sh
-
-
-# Perform any base OS specific setup
+    mkdir -p \
+        /analysis_scratch \
+        "${ANCHORE_SERVICE_DIR}"/bundles \
+        /config \
+        /home/anchore/clamav/db \
+        /licenses \
+        /var/log/anchore \
+        /var/run/anchore \
+        /workspace \
+        /workspace_preload && \
+    chown -R 1000:0 \
+        /analysis_scratch \
+        "${ANCHORE_SERVICE_DIR}" \
+        /config \
+        /home/anchore \
+        /licenses \
+        /var/log/anchore \
+        /var/run/anchore \
+        /workspace \
+        /workspace_preload && \
+    chmod -R g+rwX \
+        /analysis_scratch \
+        "${ANCHORE_SERVICE_DIR}" \
+        /config \
+        /home/anchore \
+        /licenses \
+        /var/log/anchore \
+        /var/run/anchore \
+        /workspace \
+        /workspace_preload
+
+# Install build dependencies
+RUN set -ex && \
+    yum update -y && \
+    yum module disable -y python36 && \
+    yum module enable -y python38 && \
+    yum install -y \
+        procps \
+        psmisc \
+        python38 \
+        python38-psycopg2 \
+        python38-wheel \
+        skopeo && \
+    yum clean all
+
+# Copy the installed artifacts from the first stage
+COPY --from=anchore-engine-builder /build_output /build_output
 
-# Perform the cli install into a virtual env
+# install application & all dependencies from /build_output
 RUN set -ex && \
+    echo "updating pip" && \
+    pip3 install --upgrade --no-index --find-links=/build_output/wheels/ pip && \
+    # Install anchore-cli into a virtual environment
+    echo "installing anchore-cli into virtual environment" && \
     python3 -m venv /anchore-cli && \
     source /anchore-cli/bin/activate && \
     pip3 install --no-index --find-links=/build_output/cli_wheels/ anchorecli && \
-    deactivate
-
-# Perform the anchore-engine build and install
-
-RUN set -ex && \
+    deactivate && \
+    # Install anchore-engine & required dependencies
+    echo "installing anchore-engine and required dependencies" && \
     pip3 install --no-index --find-links=/build_output/wheels/ anchore-engine && \
+    # copy all staged dependent binaries to PATH
     cp /build_output/deps/syft /usr/bin/syft && \
     cp /build_output/deps/grype /usr/bin/grype && \
     yum install -y /build_output/build_deps/*.rpm && \
-    rm -rf /build_output /root/.cache
+    # Copy default application configuration files
+    echo "copying default application config files" && \
+    cp /build_output/LICENSE /licenses/ && \
+    cp /build_output/configs/default_config.yaml /config/config.yaml && \
+    md5sum /config/config.yaml > /config/build_installed && \
+    cp /build_output/configs/docker-entrypoint.sh /docker-entrypoint.sh && \
+    chmod +x /docker-entrypoint.sh && \
+    cp /build_output/configs/clamav/freshclam.conf /home/anchore/clamav/ && \
```

Comment:
This is my fault, but we could potentially install this the same way that we do in enterprise. Meaning, we wouldn't need to copy the configuration file over to the config directory since it's already inside the anchore engine wheel.